dba0dd41613a91f17142a9bd8ea12b5abb881433..388348ddbf7d138fed3a6fe48bf6666a95ef3528

From da21c305164c3e585e29e20242afc5a31f91989f Mon Sep 17 00:00:00 2001
From: Michael Cronenworth <mike@cchtml.com>
Date: Thu, 23 Jan 2020 17:05:27 -0600
Subject: [PATCH] wbemprox: Global variable compatibility update for gcc 10.

Signed-off-by: Michael Cronenworth <mike@cchtml.com>
Signed-off-by: Alexandre Julliard <julliard@winehq.org>
---
 dlls/wbemprox/main.c             | 2 ++
 dlls/wbemprox/wbemprox_private.h | 4 ++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/dlls/wbemprox/main.c b/dlls/wbemprox/main.c
index d90f4f8cc63..09084414983 100644
--- a/dlls/wbemprox/main.c
+++ b/dlls/wbemprox/main.c
@@ -36,4 +36,6 @@ WINE_DEFAULT_DEBUG_CHANNEL(wbemprox);
 static HINSTANCE instance;
 
+struct list *table_list;
+
 typedef HRESULT (*fnCreateInstance)( LPVOID *ppObj );
 
diff --git a/dlls/wbemprox/wbemprox_private.h b/dlls/wbemprox/wbemprox_private.h
index cbf4c894f1a..cff3fe1f379 100644
--- a/dlls/wbemprox/wbemprox_private.h
+++ b/dlls/wbemprox/wbemprox_private.h
@@ -21,6 +21,6 @@
 #include "wine/list.h"
 
-IClientSecurity client_security DECLSPEC_HIDDEN;
-struct list *table_list DECLSPEC_HIDDEN;
+extern IClientSecurity client_security DECLSPEC_HIDDEN;
+extern struct list *table_list DECLSPEC_HIDDEN;
 
 enum param_direction
--
